packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1146,14 +1146,28 @@ export default class BackupNg {
         $defer.call(xapi, 'deleteVm', snapshot)
       }
 
+      const compress = getJobCompression(job)
+      if (
+        compress === 'zstd' &&
+        snapshot.$pool.restrictions.restrict_zstd_export !== 'false'
+      ) {
+        logger.warning(
+          `Zstd is not supported on the pool ${snapshot.$pool.name_label ||
```

Comment:
You can use [`?? operator`](https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator).

Reply:
I just took a look and i found that the `name_label` can be an empty string that's why i used `| |` but is not a good idea because referring to documentation the `name_label` cannot be `undefined`.